nodejs/week1/meal_sharing/src/backend/routes/large_meals.js

```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+const router = express();
+
+const meals = require("../data/meals.json");
+const review = require("../data/reviews.json");
+
+const largeMeals = meals.filter(itemMeal => {
+  if (itemMeal.price > 70) {
+    return true;
+  }
+});
+
+largeMeals.forEach(mealItem => {
+  mealItem.review = [];
```

Comment:
Please check this comment to improve your code a bit:

https://github.com/krithi-kondana/hyf-homework/pull/30#discussion_r382179162